src/modules/job-list/job-list.c

```diff
@@ -131,6 +140,11 @@ static const struct flux_msg_handler_spec htab[] = {
       .cb           = purge_cb,
       .rolemask     = 0
     },
+    { .typemask     = FLUX_MSGTYPE_REQUEST,
+      .topic_glob   = "job-list.disconnect",
+      .cb           = disconnect_cb,
+      .rolemask     = FLUX_ROLE_USER,
+    },
```

Comment:
i suppose its not a big deal, but wanted to double check you intentionally didn't want to add a "cancel stream" RPC target?  We could always add later if we need to, there's no real need for it at the moment.

Reply:
Right, no use case at the moment so I didn't add that.